docs/_docs/editor/setup.md

```diff
@@ -19,8 +19,8 @@ supported and stability is not guaranteed.
 
 ## Quick Install
 
-Assuming you have met all the prerequisites for your platform, culminating with **Atom 1.7.4 or
-greater** being installed, the easiest way to install Nuclide is either at the command line:
+Assuming you have met all the prerequisites for your platform, culminating with *Atom 1.10.2 or
```

Comment:
Wanted to get your thoughts here...

Instead of pointing to a specific Atom version that will change over time, does it make sense to say something like:

"culminating with a version of Atom that is equal to or greater than that specified in [Atom's dependencies under "atom"](https://github.com/facebook/nuclide/blob/master/package.json)"

Or is that too much?

I am just thinking that way we don't have to continually update it.